dags/main_summary.py

```diff
@@ -21,9 +21,23 @@
     'retry_delay': timedelta(minutes=30),
 }
 
+default_args_weekly = {
+    "owner": "frank@mozilla.com",
+    "depends_on_past": False,
+    "start_date": datetime(2019, 5, 30),
+    "email": ["telemetry-alerts@mozilla.com", "frank@mozilla.com"],
+    "email_on_failure": True,
+    "email_on_retry": True,
+    "retries": 2,
+    "retry_delay": timedelta(minutes=30),
+}
+
 # Make sure all the data for the given day has arrived before running.
 # Running at 1am should suffice.
 dag = DAG('main_summary', default_args=default_args, schedule_interval='0 1 * * *')
+dag_weekly = DAG(
+    "weekly_main_summary", default_args=default_args_weekly, schedule_interval="@weekly"
+)
```

Comment:
I'd probably move this out into a separate file like the hardware report: https://github.com/mozilla/telemetry-airflow/blob/master/dags/fx_usage_report.py

There's no hard dependency on the daily main_summary case here.